lib/workos/sso.rb

```diff
@@ -119,6 +119,35 @@ def profile(code:, project_id:)
         WorkOS::Profile.new(response.body)
       end
 
+      # Promote a DraftConnection created via IdP Link Embed such that the
+      # Enterprise users can begin signing into your application.
+      #
+      # @param [String] token The draft connection token that's been provided to
+      # you by the IdP Link Embed
+      #
+      # @example
+      #   WorkOS::SSO.promote_draft_connection(
+      #     token: 'draft_conn_429u59js',
+      #   )
+      #   => true
+      #
+      # @return [Bool] - returns `true` if successful, `false` otherwise.
+      # @see https://github.com/workos-inc/idp-link
+      sig do
+        params(
+          token: String,
+        ).returns(T::Boolean)
+      end
+      def promote_draft_connection(token:)
```

Comment:
Can we update this to `activate_draft_connection`?

And something other than `token` since it's actually the resource id. `draft_connection_id`?

Reply:
that breaks so much documentation etc. we've always said we were gonna call it a token